contracts/Rivendale.sol

```diff
@@ -62,16 +62,32 @@ contract Rivendale {
         uint256 _walker,
         uint256 _user
     ) internal {
+        require(
+            _trb.add(_user).add(_walker) == 1000,
+            "weights must sum to 1000"
+        );
         weights.trbWeight = _trb;
         weights.userWeight = _user;
         weights.walkerWeight = _walker;
     }
 
-    function getWeights() external view returns(uint256,uint256,uint256){
-        return (weights.trbWeight,weights.userWeight,weights.walkerWeight );
+    function getWeights()
+        external
+        view
+        returns (
+            uint256,
+            uint256,
+            uint256
+        )
+    {
+        return (weights.trbWeight, weights.userWeight, weights.walkerWeight);
     }
 
+    //Should we limit the amount of open votes at a given time?

```

Comment:
thoughts?  I think we make it more flexible, but is there a scenario where the funds break if both want to slash?